packages/tests/integration/src/EntityManager.reactiveRules.test.tsx

```diff
@@ -210,60 +210,65 @@ describe("EntityManager.reactiveRules", () => {
   });
 
   it.withCtx("creates the right reactive rules", async () => {
-    const cstr = expect.any(Function);
     const fn = expect.any(Function);
-    expect(getMetadata(Author).config.__data.reactiveRules).toEqual([
+    expect(getReactiveRules(Author)).toMatchObject([
       // Author's firstName/book.title validation rule
-      { cstr, name: sm(/Author.ts:\d+/), fields: ["firstName"], path: [], fn },
+      { cstr: "Author", name: sm(/Author.ts:\d+/), fields: ["firstName"], path: [], fn },
```

Comment:
This is just updating all of the assertions to use this `getReactiveRules` helper methods that maps the constructors over to `cstr.name` because otherwise the jest output, as usual, is terribly huge/large/dumb.